scripts/build_test.py

```diff
@@ -892,7 +927,7 @@ def mock_build_using_webpack(config_path):
         def mock_ensure_files_exist(unused_filepaths):
             check_function_calls['ensure_files_exist_gets_called'] = True
 
-        def mock_modify_constants(unused_prod_env, unused_maintenance_mode):
+        def mock_modify_constants(unused_prod_env, maintenance_mode):  # pylint: disable=unused-argument
```

Comment:
Any specific reason for this changes? (We try to avoid adding pragmas*)

Reply:
Since `maintenance_mode` is a keyword argument it cannot change name, because then the call to that method would not work.